src/sidebar/components/Thread.js

```diff
@@ -187,9 +158,17 @@ function Thread({ thread, threadsService }) {
 
         {countHidden(thread) > 0 && (
           <div className="space-y-2">
-            {!thread.parent && (
-              <HiddenThreadCardHeader
+            {/**
+             * Every ThreadCard should have a header of some sort representing
+             *  its top-level thread. When a thread is visible, an Annotation
+             *  is rendered which in turn renders an AnnotationHeader. But when
+             *  a thread is hidden, Annotation is not rendered: reander a header
+             *  here instead for hidden top-level threads.
+             */}
+            {!thread.parent && thread.annotation && (
```

Comment:
```suggestion
            {
              // Every ThreadCard should have a header of some sort representing
              // its top-level thread. When a thread is visible, an Annotation
              // is rendered which in turn renders an AnnotationHeader. But when
              // a thread is hidden, Annotation is not rendered: reander a header
              // here instead for hidden top-level threads.
              !thread.parent && thread.annotation && (
```

This isn't a JSDoc comment, so no need for `/**`.